File: packages/core/frontend/src/types/resources/resourcequota.ts
export interface ResourceQuota {
  Name: string;
  Namespace: string;
  Age: string;
}

export interface ResourceQuotaDetail {
  Name: string;
  Namespace: string;
  Age: string;
  CreatedAt: string;
  Labels: Record<string, string>;
  Annotations: Record<string, string>;
  Hard: Record<string, string>;
  Used: Record<string, string>;
}

File: packages/core/kube/dto/resourcequota.go
package dto

type ResourceQuotaRef struct {
	Namespace string `json:"namespace"`
	Name      string `json:"name"`
}

type ResourceQuota struct {
	Name      string
	Namespace string
	Age       string
}

type ResourceQuotaDetail struct {
	Name        string
	Namespace   string
	Age         string
	CreatedAt   string
	Labels      map[string]string
	Annotations map[string]string
	Hard        map[string]string
	Used        map[string]string
}
